static/src/javascripts/projects/common/modules/commercial/commercial-feature-policies.js

```diff
@@ -53,6 +55,17 @@ define([
         }
     };
 
+    policies.membershipMessages = function () {
+        if (!detect.adblockInUse() &&
+            detect.getBreakpoint() !== 'mobile' &&
+            config.page.contentType === 'Article'
+        ) {
```

Comment:
I'd like to move the US version of this message into `membership-messages` too, so the `UK` check is fairly circumstantial. I've just moved the shared rules into here.